src/api.ts

```diff
@@ -85,6 +87,74 @@ function getCollectKeyFromCommand(command: string): string|undefined {
   }
 }
 
+/**
+ * Validate the request.
+ *
+ * Throws if `params` are invalid for `command`.
+ */
+function _validateRequest(command: string, params: any): void {
```

Comment:
Remove this function, unnecessary/redundant to the validation rippled is already doing (see last comment here: https://github.com/ripple/ripple-lib/pull/865#discussion_r174861916)